ui/components/app/srp-quiz-modal/SRPQuiz/SRPQuiz.tsx

```diff
@@ -24,10 +24,10 @@ import {
   IconName,
   IconSize,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 import QuizContent from '../QuizContent';
 import { JSXDict, QuizStage } from '../types';
 
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 46 06 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/36dacec7-5188-4c76-a92e-a78af58a9cd3">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 46 17 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/965efe86-08a9-4055-a8ee-0fd63f6bc2d2">